docs/UserGuide.md

```diff
@@ -101,7 +103,7 @@ Examples:
 
 ### Adding a property to buy: `addBuy`
 
-Adds a property which has to be bought to the address book. The property is associated to a contact in the Address Book. 
+Adds a property which has to be bought to the address book. The property is associated to a contact in the Address Book.
```

Comment:
Should it be 'wants to buy' instead of 'has to be bought'? Because just because he adds the property doesn't mean he is guaranteed to buy it but rather he is just interested in it?